packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -0,0 +1,163 @@
+import { Input } from '@botonic/core'
+import axios from 'axios'
+
+import {
+  HtFallbackNode,
+  HtFlowBuilderData,
+  HtIntentNode,
+  HtKeywordNode,
+  HtNodeComponent,
+  HtNodeStartType,
+  HtNodeWithContent,
+  HtNodeWithContentType,
+  HtStartNode,
+} from './content-fields/hubtype-fields'
+import { FlowBuilderApiOptions } from './types'
+
+export class FlowBuilderApi {
+  url: string
+  flow: HtFlowBuilderData
+
+  private constructor() {}
+
+  static async create(options: FlowBuilderApiOptions): Promise<FlowBuilderApi> {
+    const newApi = new FlowBuilderApi()
+
+    newApi.url = options.url
```

Comment:
`newApi` sounds strange to me, what about renaming it to just api? as it's a local variable no need to be "new", IMHO.